tst/EnergyPlus/unit/PlantCondLoopOperation.unit.cc

```diff
@@ -775,6 +782,118 @@ TEST_F(DistributePlantLoadTest, DistributePlantLoad_SequentialUniformPLR)
     EXPECT_EQ(remainingLoopDemand, 10.0);
 }
 
+TEST_F(DistributePlantLoadTest, DistributePlantLoadSequentialDryBulbRB)
```

Comment:
New unit tests, based on `DistributePlantLoad_Sequential`, that check if plant loop load is passed to components only if they are allowed to operate based on the outdoor air (and reference node) conditions.